backend/common/src/main/java/ai/verta/modeldb/common/db/migration/Migrator.java

```diff
@@ -16,6 +19,18 @@ public Migrator(Connection connection, String resourcesDirectory) {
     this.resourcesDirectory = resourcesDirectory;
   }
 
+  /** Assumes that the underlying database has already been created. */
+  public void performMigration(RdbConfig config) throws SQLException, MigrationException {
```

Comment:
the execute migration method is just for a single migration (and is non-public), rather than "perform" which is the whole orchestration of migrations.

Reply:
I'm going to rename the "execute" one to "executeSingleMigration" downstream, to make it a bit more clear what's up.